This patch requires that we also enable building of the
texinfo-native recipe which will install the makedoc tool 
for the host machine.

This patch simply uses the newly installed makedoc tool from
sysroot.

Upstream-Status: Inappropriate [OE-Specific]

Signed-off-by: Saul Wold <sgw@linux.intel.com>

Index: texinfo-4.13/info/Makefile.am
===================================================================
--- texinfo-4.13.orig/info/Makefile.am	2008-05-22 05:11:33.000000000 -0700
+++ texinfo-4.13/info/Makefile.am	2011-12-10 12:55:53.604440118 -0800
@@ -75,7 +75,7 @@
 # more than once.
 funs.h: makedoc$(EXEEXT) $(cmd_sources)
 	rm -f $(generated_sources)
-	$(top_builddir)/$(native_tools)/info/makedoc $(cmd_sources)
+	makedoc $(cmd_sources)
 
 # The following hack is necessary to hint make before the automatic
 # dependencies are built.
Index: texinfo-4.13/doc/Makefile.am
===================================================================
--- texinfo-4.13.orig/doc/Makefile.am	2008-09-18 11:31:56.000000000 -0700
+++ texinfo-4.13/doc/Makefile.am	2011-12-10 13:04:09.216457601 -0800
@@ -19,7 +19,7 @@
 
 # Use the programs built in our distribution, taking account of possible
 # cross-compiling.
-MAKEINFO = $(top_builddir)/$(native_tools)/makeinfo/makeinfo
+MAKEINFO = makeinfo
 
 # We'd also like to use something like this, but Automake calls
 # "install-info" directly.
